airflow/www/static/js/tree/details/Header.jsx

```diff
@@ -72,22 +73,22 @@ const Header = () => {
   const isTaskDetails = runId && taskId;
 
   return (
-    <Breadcrumb color="gray.300" mt={4}>
+    <Breadcrumb mt={4} separator={<Text color="gray.300">/</Text>}>
```

Comment:
The separator was `/` before? (Just wondering where the added `<Text>` node is needed

Reply:
The Text node allows me to style just the `/` with `gray.300` instead of every child element and then having to manually set children back to a darker color.

Reply:
I'm not exactly sure what your compile markup looks like, but I'd guess you don't actually need/want a `<p></p>`?
```suggestion
    <Breadcrumb mt={4} separator={<Box as="span" color="gray.300">/</Box>}>
```

Reply:
Where was the `/` before? (I saw it in the screenshots, but not the markup)

Reply:
It was auto-generated by the component before. This just replaced the default.